docs/GettingStarted.md

```diff
@@ -152,65 +154,75 @@ but will consider them as having low priority.
 
 ## Installation
 
-The following steps will help you quickly start tracing your Ruby application.
+Adding tracing to your Ruby application only takes a few quick steps:
 
-### Configure the Datadog Agent for APM
+1. Setup the Datadog Agent for tracing
+2. Instrument your application
+3. Connect your application to the Datadog Agent
 
-Before downloading tracing on your application, [install the Datadog Agent on the host](https://docs.datadoghq.com/agent/). The Ruby APM tracer sends trace data through the Datadog Agent.
+### Setup the Datadog Agent for tracing
 
-Install and configure the Datadog Agent to receive traces from your now instrumented application. By default the Datadog Agent is enabled in your `datadog.yaml` file under `apm_enabled: true` and listens for trace traffic at `localhost:8126`. For containerized environments, follow the steps below to enable trace collection within the Datadog Agent.
+Before installing `ddtrace`, [install the Datadog Agent](https://docs.datadoghq.com/agent/), to which `ddtrace` will send trace data.
 
-#### Containers
+Then configure the Datadog Agent to accept traces. To do this, either:
```

Comment:
Coincidence, turns out I had a Q today WRT agent configuration where this section seemed to imply it was tracer config via `datadog.yaml`, the new wording and structure makes it a bit clearer that this is agent related. I'm wondering if we can improve this further.